macros/impl/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "foundry-macros-impl"
```

Comment:
how about renaming this to `-derive`? instead of `-impl`?

Reply:
I was thinking about that, but then that also implies new crates for attributes and normal proc macros, so I don't know

Reply:
I see, that makes sense.